src/main/scala/com/mwz/sonar/scala/ScalaPlugin.scala

```diff
@@ -118,7 +125,12 @@ final class ScalaPlugin extends Plugin {
       // Scoverage
       classOf[ScoverageMetrics],
       classOf[ScoverageReportParser],
-      classOf[ScoverageSensor]
+      classOf[ScoverageSensor],
+      // Scapegoat
```

Comment:
We should add a scapegoat module test [here](https://github.com/mwz/sonar-scala/blob/master/src/test/scala/com/mwz/sonar/scala/ScalaPluginSpec.scala) too.

_**PS:** I forgot to add this in the review._

Reply:
Oh yes, true!